web/src/components/filings-calendar/FilingsCalendar.test.tsx

```diff
@@ -37,8 +37,7 @@ import {
 import * as materialUi from "@mui/material";
 import { createTheme, ThemeProvider, useMediaQuery } from "@mui/material";
 import { fireEvent, render, screen, within } from "@testing-library/react";
-import dayjs from "dayjs";
-
+import dayjs, { Dayjs } from "dayjs";
```

Comment:
[dust] We don't use it very much across the codebase (really I only found two instances, `serverless.ts` and `basicauth.ts`), but I'd like for us to get in the habit of using `import type` when importing types. So this line would look like:

```typescript
import dayjs, { type Dayjs } from "dayjs";
```

As a side note, in newer codebases, you might see something like this, but I'd advise against this pattern because we don't use it anywhere else in our code:

```typescript
import { default as dayjs, type Dayjs } from "dayjs";
```

Reply:
Interesting, didn't know about this. Will do in the future